jsk_kinova_robot/README.md

```diff
@@ -71,11 +71,12 @@ wstool init
 wstool merge https://raw.githubusercontent.com/jsk-ros-pkg/jsk_robot/master/jsk_kinova_robot/kinova.rosinstall
 wstool update
 cd ../
-source /opt/ros/melodic/setup.bash
+source /opt/ros/$ROS_DISTRO/setup.bash
 rosdep install -y -r --from-paths src --ignore-src
 catkin build jsk_kinova_startup kinovaeus
 source devel/setup.bash
 ```
+If an error related to `collada_urdf` occurs during build in noetic environments, it may be solved by referring to https://github.com/jsk-ros-pkg/jsk_robot/issues/1892 .
```

Comment:
Please adopt https://github.com/jsk-ros-pkg/jsk_robot/issues/1892 to rosinstall and write the comment like `# We can remove this line after the https://github.com/jsk-ros-pkg/jsk_robot/issues/1892 is merged` . Rewriting root package area is not recommended.

Reply:
Are you talking about the https://github.com/ros/collada_urdf/pull/44 's PR to be adopted in rosinstall?
And like https://github.com/ojh6404/informatized_body_project/pull/2, with further comments(`# We can remove this line after the https://github.com/ros/collada_urdf/pull/44 is merged`)?

Reply:
Yes

Reply:
Thank you! I fixed it. https://github.com/jsk-ros-pkg/jsk_robot/pull/1906/commits/70cacaf048271454a4ed896b6cf26e6b05de2452